static/src/javascripts/types/global.d.ts

```diff
@@ -79,34 +79,35 @@ interface Config {
 type Edition = string; // https://github.com/guardian/frontend/blob/b952f6b9/common/app/views/support/JavaScriptPage.scala#L79
 
 interface PageConfig extends CommercialPageConfig {
```

Comment:
I have alphabetised this list, to make it easy to find what is there. If there was a reason why this list was intentionally not alphabetised, please let me know and I will change it back